tv_apps/dashboard/js/dashboard.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+/* global KeyNavigationAdapter, Dashboard */
+
+(function(exports) {
+
+  function Dashboard () {}
+
+  Dashboard.prototype = {
+
+    init: function () {
+      // #main-section DOM reference
+      this.mainSection = document.getElementById('main-section');
+      this.mainSection.dataset.activeDirection = '';
+      this.mainSection.focus();
+
+      this.keyNavigationAdapter = new KeyNavigationAdapter();
+      this.keyNavigationAdapter.on('move', this.onMove.bind(this));
+      this.keyNavigationAdapter.init(this.mainSection);
+    },
+
+    onMove: function (key) {
+      var activeDirection = this.mainSection.dataset.activeDirection;
+
+      switch (activeDirection) {
+        case 'up':
+          if (key === 'down') {
```

Comment:
Although it makes sense to me that pressing the opposite direction to move back but the UX spec said we should do that by pressing the key again or pressing back key. May you confirm this with UX team?